C/manual/ch_Finance-Quote.xml

```diff
@@ -332,88 +337,122 @@ SYNOPSIS
-        Finance::Quote fields GnuCash uses:
-            symbol: AAPL            &lt;=== required
-              date: 10/14/2022      &lt;=== recommended
-          currency: USD             &lt;=== required
-              last: 138.38          &lt;=\
-               nav:                 &lt;=== one of these
-             price:                 &lt;=/
-      </screen>
-                  </informalexample>
-                </para>
-              </listitem>
-
-              <listitem>
-                <para>With the <userinput>-v</userinput> option a possibly longer output showing all of the fields
-                  &app-fq; returned. This can be useful to troubleshoot problems with a &app-fq;
-                  source module.
-                  <informalexample>
+  $ gnucash-cli --quotes dump yahoo_json AAPL
+  Finance::Quote fields GnuCash uses:
+      symbol: AAPL            &lt;=== required
+        date: 10/14/2022      &lt;=== recommended
+    currency: USD             &lt;=== required
+        last: 138.38          &lt;=\
+         nav:                 &lt;=== one of these
+       price:                 &lt;=/
+</screen>
+                    </informalexample>
+                  </para>
+                </listitem>
+
+                <listitem>
+                  <para>With the <userinput>-V</userinput> option a possibly longer output showing all of the fields
+                    &app-fq; returned. This can be useful to troubleshoot problems with a &app-fq;
+                    source module.
+                    <informalexample>
 <?dbfo pgwide="1"?>
 <screen language="console">
-        $ ALPHAVANTAGE_API_KEY=123456789 bin/gnucash-cli -V --quotes dump alphavantage INTC
-        INTC:
-                     method =&gt; alphavantage
-                   p_change =&gt; -1.9304
-                       high =&gt; 26.6300
-                       date =&gt; 10/14/2022
-         currency_set_by_fq =&gt; 1
-                      close =&gt; 26.4200
-                   currency =&gt; USD
-                    isodate =&gt; 2022-10-14
-                        low =&gt; 25.7600
-                    success =&gt; 1
-                        net =&gt; -0.5100
-                       open =&gt; 26.4600
-                     symbol =&gt; INTC
-                     volume =&gt; 48118005
-                       last =&gt; 25.9100
-      </screen>
-                  </informalexample>
-                  Notice that in this case we used alphavantage and provided the
-                  <userinput>ALPHAVANTAGE_API_KEY</userinput> on the command line. That's not
-                  necessary if the key is already stored in the shell environment or in GnuCash
-                  preferences.
-                </para>
-              </listitem>
-            </itemizedlist>
-          </listitem>
-        </itemizedlist>
+  $ ALPHAVANTAGE_API_KEY=123456789 bin/gnucash-cli -V --quotes dump alphavantage INTC
+  INTC:
+               method =&gt; alphavantage
```

Comment:
`=&gt` is the ASCII representation of `&Implies;`, but I am not sure that it was intended by the Python designers.
For `&gt=` I have usually the reflex 'replace it by `&ge;`.